ee/ucp/admin/install/cloudproviders/install-on-azure-custom.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Custom Azure Roles
+description: Learn how to create custom RBAC roles to run Docker Enterprise on Azure.
+keywords: Universal Control Plane, UCP, install, Docker Enterprise, Azure, Swarm
+---
+
+## Overview
+
+This document describes how to create custom roles to deploy Docker Enterprise resources.
```

Comment:
Should we add Azure in the overview section for clarity? That is,`This document describes how to create custom Azure roles...`?